Source/WebKit/NetworkProcess/webrtc/NetworkRTCTCPSocketCocoa.h

```diff
@@ -29,6 +29,7 @@
 
 #include "NetworkRTCProvider.h"
 #include <Network/Network.h>
+#include <wtf/NativePromise.h>
```

Comment:
```suggestion
#include <wtf/Forward.h>
```

NativePromise.h can be included in the code only.

Reply:
done